### PR TITLE
Forward redis.publish's responses

### PR DESCRIPTION
**Pull request by Amoki**
publish method of python redis implementation returns the number of clients that have got the message.

Forward this as publish_message's return might be usefull for some reasons, especially for the issue https://github.com/jrief/django-websocket-redis/issues/129

### CODE REVIEW

**Comment by Eirika** [comment]
:+1:  as it totally solve my problem ! :)

**Comment by jrief** [comment]
@Amoki : this PR perfectly makes sense.
Could you please add or modify a unit test to check your code against possible future regressions.

This change of beahvior should also be documented.

**Comment by Amoki** [comment]
@jrief Updated.
Tell me if a missed some contibuting rules !

File: docs/usage.rst [hunk 1]
```diff
@@ -189,9 +189,30 @@ now, the message “Hello World” is sent to all clients using the session key
 
 In this context the the magic item ``SELF`` refers to all clients owning the same session key.
 
+Get informations about published messages
```
Comment (roidelapluie) [inline docs/usage.rst]: information without s ? or rephrase?

**Comment by jrief** [comment]
@Amoki do you have any idea why your test ``test_number_of_receivers `` fails on Django-1.7.11 and Django-1.8.8 ?

**Comment by Amoki** [comment]
Absolutly no idea :disappointed: 
It appears to be random. If I launch a lot of test locally, I can reproduce it !
I'll have to go deeper.

**Pull request closed without merging**